### DISCUSSION TITLE
Any update for the overall corona-warn-app project regarding log4j / log4shell?

### DISCUSSION THREAD

**Discussion by Discussion-008** [corona-warn-app/cwa-server#1654] [Q&A] [answered]
Can anybody give an update regarding the **log4j / log4shell vulnerabilty** of any of the corona-warn-app software parts? Is any of the project parts affected? If affected at all, are the required updates are already in place and are the servers updated?

**Comment by Ein-Tim** [comment, marked as answer]
Hi @Discussion-008

There is a FAQ entry regarding this: [Statement regarding the log4j vulnerability](https://www.coronawarn.app/en/faq/#log4j).

It says that log4j was updated to version 2.16.0 & you can take a look at the merged PRs in this repo to confirm that this actually happened.

**Reply by Discussion-008** [reply]
Thanks for the quick reply

**Reply by dsarkar** [reply]
@Discussion-008 Thanks for participating here. If you are satisfied for now, please, close the issue. Of course, if there are any further questions, suggestions, reports, etc open an issue in the corresponding repository.

---
Corona-Warn-App Open Source Team

**Reply by Ein-Tim** [reply]
@dsarkar Can discussions be "closed"? I think marking them as answered is all you can do, if I'm not mistaken.
